### ISSUE TITLE
Hyperparametzer for deepset/bert-base-german-cased-sentiment-Germeval17

### ISSUE THREAD

**Issue by PhilipMay** [deepset-ai/FARM#435]
**Question**
You provided a model called [deepset/bert-base-german-cased-sentiment-Germeval17](https://huggingface.co/deepset/bert-base-german-cased-sentiment-Germeval17). Could you share the hyperparameters you used for training? Maybe in form of a blog entry?

**Additional context**
I am tryingto fit `bert-base-german-cased` to Germeval17 but did not reach your performance (F1 or Acc). I just changed the batch size, gradient_accumulation_steps and learning rate. What else is useful to change in an hyperparameter search?

Thanks
Philip

**Comment by bogdankostic** [comment]
There is already a blog article explaining how we trained this particular model and which hyperparameters we used. You can find it here: https://towardsdatascience.com/using-transformer-based-language-models-for-sentiment-analysis-dc3c10261eec

Hope this will help you! :)

**Comment by PhilipMay** [comment]
Great thanks!
